app/views/search/search_boxed.html.erb

```diff
@@ -40,12 +40,12 @@
     </div>
     <div class="col1q-r">
     <div class="bit">
-      <h3>Supporting content</h3>
-      <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr: </p>
+      <h3>Other options</h3>
+      <p>If you're not finding what you need, you can try one of these: </p>
       <ul>
-        <li class=""><a href="">Sed diam nonumy eirmod</a></li> 
-        <li class=""><a href="">Tempor invidunt ut labore</a></li> 
-        <li class=""><a href="">Dolore magna aliquyam</a></li>
+        <li class="http://libraries.mit.edu/bartonplus"><a href="library.mit.edu">Search in BartonPlus</a></li> 
```

Comment:
... and this is why we have code reviews *before* merge. 😞  and yes, we should be linking with https. I'll make any changes in the next PR.